src/style.css

```diff
@@ -1,3 +1,99 @@
+/* * {
+  text-decoration: none;
+  margin: 0;
+  padding: 0;
+} */
+
 body {
   margin: 0;
+  /* padding: 0;
+  font-family: "Roboto", sans-serif; */
+}
+
+html {
+  /* background-color: gray; */
+  --apple-width: 37px;
+  /* margin: 0;
+  padding: 0; */
+}
+
+.header {
+  background-color: #fff;
+  /* width: 924px; */
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  text-align: center;
+  margin: 0 auto;
+}
+
+.nav {
+  margin: 10px;
+}
+
+.nav__img {
+  height: 40px;
+  width: 40px;
+  margin: 10px 0;
+  position: relative;
+  top: -5px;
+}
+
+.nav__link {
+  color: #000;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  /* margin: 0 10px;
+  padding: 10px 5px; */
+  text-decoration: none;
+  padding: 2em 5px;
+  margin: -2em 5px;
```

Comment:
Check the mockup it says what paddings you should have, just don't write them in float format, round them up